ios/private/discover_ios_targets.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+# Script to help discover all iOS targets that should be included in ios_spm_package
+# This analyzes your entire codebase and suggests what to include
+#
+# Generated with Cursor by Koriann South on August 29, 2025
+#
+# Usage:
+#   bazel run //:discover-ios-targets
+#   OR
+#   ./helpers/release/discover-ios-targets.sh
+#
+# Manual Discovery Commands:
+#   # Find all Swift source targets:
+#   bazel query "//..." | grep "_Sources$" | grep -E "(swiftui|ios)"
+#
+#   # Find all JS native bundles:
+#   bazel query "//..." | grep ":core_native_bundle$"
+#
+#   # Find all targets in workspace:
+#   bazel query "//..."
+
+set -euo pipefail
+
+# Check if output file argument is provided
+if [[ $# -lt 1 ]]; then
+    echo "Usage: $0 <output_file> [variable_name]"
+    echo "Example: $0 ios_targets.bzl"
+    echo "Example: $0 ios_targets.bzl my_sources"
+    exit 1
+fi
+
+OUTPUT_FILE="$1"
+VARIABLE_NAME="${2:-sources}"  # Default to "sources" if not provided
+
+echo "🔍 Discovering all iOS targets for ios_spm_package..."
+echo "📝 Output file: $OUTPUT_FILE"
+echo "🏷️  Variable name: $VARIABLE_NAME"
+echo ""
+
+if [[ -n "${BUILD_WORKSPACE_DIRECTORY:-}" ]]; then
+    # Running through bazel run - change to workspace directory
+    cd "$BUILD_WORKSPACE_DIRECTORY"
+fi
+
+# Find all Swift source targets across the entire workspace
+echo "📱 All Swift Source Targets:"
+echo "============================"
+ALL_SWIFT_SOURCES=$(bazel query "//..." 2>/dev/null | grep "_Sources$" | grep -E "(swiftui|ios)" | sort)
```

Comment:
Could we bake the filtering into the query itself? This example filters on naming:
https://github.com/player-ui/player/blob/9439eff801c9c9441e00c63b9572e42420c4edcb/justfile#L29

Reply:
Done